draft-vvv-tls-cross-sni-resumption.md

```diff
@@ -124,6 +130,14 @@ for that certificate.  In addition, creating a certificate that is valid for
 multiple hostnames can lead to complications if some of those hostnames change
 ownership, or otherwise require a different operational domain.
 
+Session tickets can contain arbitrary information, and thus could be
+potentially used to re-identify a user from a previous connection.
+Cross-domain resumption expands the potential list of servers to which an
+individual ticket could be presented.  Mitigation of this potential privacy
+issue depends on the specifics of the application using TLS.  For the Web use
```

Comment:
I wonder if we can say something more specific, since I think partitioning is generally the right strategy here.  I wrote something about partitioning in https://tools.ietf.org/html/draft-ietf-tls-batch-signing-00#section-4.4.

Perhaps something like:

Client applications should partition the session cache between connections that are meant to be uncorrelated. For example, the Web use case uses network partition keys to separate cache lookups [FETCH].

Reply:
Thanks!  Added that instead.